db/migrate/20160417201140_create_response_groups.rb

```diff
@@ -0,0 +1,10 @@
+class CreateResponseGroups < ActiveRecord::Migration
+  def change
+    create_table :response_groups do |t|
+      t.references :brand, index: true, foreign_key: true
```

Comment:
Any reason we're tying this to brand? We can get that through the signal if needed.